src/shared/utils.ts

```diff
@@ -188,4 +200,54 @@ export class Utils {
       setTimeout(resolve, time);
     });
   }
+
+  /**
+   * Spawn a Node child process with predefined environment variables
+   * @param command CLI Command - NO ARGS
+   * @param spawnArgs Array of arguments for CLI command
+   */
+  public static spawn(options: ServerlessSpawnOptions): Promise<void> {
+    const { serverless, commandArgs, onSigInt } = options;
+    const { command } = options;
+    // Run command from local node_modules
+    let localCommand = join(
+      serverless.config.servicePath,
+      "node_modules",
+      ".bin",
+      command
+    );
+    
+    // Append .cmd if running on windows
+    if (process.platform === "win32") {
+      localCommand += ".cmd";
+    }
+
+    const env = {
+      // Inherit environment from current process, most importantly, the PATH
+      ...process.env,
+      // Environment variables from serverless config are king
+      ...serverless.service.provider["environment"],
+    }
+    if (!options.silent) {
+      serverless.cli.log(`Spawning process '${command} ${commandArgs.join(" ")}'`);
```

Comment:
nit: we should refactor the log method to `log`, `info`, etc. and turn each on/off accordingly with the `silent` flag

Reply:
Yep, that's already in the [backlog](https://github.com/serverless/serverless-azure-functions/issues/374)

Reply:
Although this `log` method is part of the serverless core, not something we can refactor.